.changeset/honest-elephants-care.md

```diff
@@ -0,0 +1,5 @@
+---
+"@ebay/ui-core-react": patch
```

Comment:
Seems more like a feature, so Id put it in a minor version

Reply:
Technically I don't need a version for this, it was just to test the action

Reply:
I will remove this changeset. It is a `core` change that doesn't need a version update